src/styles.js

```diff
@@ -11,3 +11,6 @@ require('@vars/zindex.css');
 // Import root and richtext CSS
 require('@atoms/Root/Root.css');
 require('@atoms/RichText/RichText.css');
+
+// Third party libs
+require('tiny-slider/dist/tiny-slider.css');
```

Comment:
Lets move this to the Carousel.css to keep these as close to the context in which its being used:
```css
@import "tiny-slider/dist/tiny-slider.css";
```